ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -97,6 +97,13 @@ MiCreatePebOrTeb(IN PEPROCESS Process,
         return STATUS_NO_MEMORY;
     }
 
+    Status = PsChargeProcessNonPagedPoolQuota(PsGetCurrentProcess(), sizeof(MMVAD_LONG));
```

Comment:
`MiCreatePebOrTeb` takes a `Process`, so you can just use that as the target process to charge its quota.